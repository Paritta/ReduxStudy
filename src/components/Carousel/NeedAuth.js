import React from "react";
import styled from "styled-components";
import oc from "open-color";

const propTypes = {
};

const defaultTypes = {
};

const Wrapper = styled.div`
    background: white;
    height: 50vh;
    
    display: flex;
    align-items: center;
    justify-content: center;
`;

const Span= styled.div`
    color: ${oc.gray[5]};
    font-family: 'Hanna', fantasy;
    font-size: 3em;
`;

export class NeedAuth extends React.Component {
    render () {

        return (
            <Wrapper>
                <Span>
                    로그인이 필요합니다.
                </Span>
            </Wrapper>
        )
    }
}

NeedAuth.propTypes = propTypes;
NeedAuth.defaultTypes = defaultTypes;

export default NeedAuth;
