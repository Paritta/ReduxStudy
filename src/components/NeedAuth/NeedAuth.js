import React from "react";
import styled from "styled-components";

const propTypes = {
};

const defaultTypes = {
};

const Wrapper = styled.div`
    background: hsla(0, 100%, 0%, 0.4);
    height: 50vh;
    
    display: flex;
    align-items: center;
    justify-content: center;
    
    width: 100%;
`;

const Span= styled.div`
    color: white;
    font-family: 'Hanna', fantasy;
    font-size: 3em;
`;

export class NeedAuth extends React.Component {
    render () {

        return (
            <Wrapper>
                <Span>
                    로그인이 필요합니다.
                </Span>
            </Wrapper>
        )
    }
}

NeedAuth.propTypes = propTypes;
NeedAuth.defaultTypes = defaultTypes;

export default NeedAuth;
