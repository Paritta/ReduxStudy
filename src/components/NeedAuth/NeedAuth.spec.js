import React from "react";
import { NeedAuth } from "./NeedAuth";
import "jest-styled-components";
import ReactTestRenderer from "react-test-renderer";

it("Card render correctly", () => {
    const tree = ReactTestRenderer.create(
        <NeedAuth />
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
