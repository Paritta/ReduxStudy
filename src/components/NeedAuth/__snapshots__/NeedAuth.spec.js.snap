// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render correctly 1`] = `
.c0 {
  background: hsla(0,100%,0%,0.4);
  height: 50vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
}

.c1 {
  color: white;
  font-family: 'Hanna',fantasy;
  font-size: 3em;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    로그인이 필요합니다.
  </div>
</div>
`;
